src/app/components/SelectColor.js

```diff
@@ -0,0 +1,16 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+export default class SelectColor extends React.Component {
+  render() {
+    return (
+      <select id={this.props.id} value={this.props.value} onChange={ this.props.onChange } >
```

Comment:
i would move those options into an array, and then render those inside, using .map() function

https://reactjs.org/docs/lists-and-keys.html#rendering-multiple-components